AUTHORS

```diff
@@ -0,0 +1,3 @@
+Thomas Oster
+
+Google Inc.
```

Comment:
I'm contributing while working as part of my 20% non-primary-project time.
I'm interested in it because I don't want the proprietary windows-only laser driver software.